pkg/pillar/agentlog/parsejson.go

```diff
@@ -10,12 +10,15 @@ import (
 // Extend this structure with optional specific tags from
 // log.WithFields
 type Loginfo struct {
-	Level    string `json:"level"`
-	Time     string `json:"time"` // RFC3339 with Nanoseconds
-	Msg      string `json:"msg"`
-	Function string `json:"func"`
-	Filename string `json:"file"`
-	Source   string `json:"source"`
+	Level         string `json:"level"`
+	Time          string `json:"time"` // RFC3339 with Nanoseconds
+	Msg           string `json:"msg"`
+	Function      string `json:"func"`
+	Filename      string `json:"file"`
+	Source        string `json:"source"`
+	Appuuid       string `json:"appuuid"`
+	Containername string `json:"containername"`
+	Eventtime     string `json:"eventtime"`
```

Comment:
Is Time different from Eventime? Is Eventime RFC3339?
Does Brazos in cloud know about Eventtime?

Brazos does not have to care if Log Time and Eventtime signify different events. But, I just would like to understand more.

Reply:
@gkodali-zededa the Time is when 'zedcloud' gets the log and log into the system, 'eventtime' is when container did the logging before we fetch them. They are different. Eventtime is RFC3339 format. This eventtime actually is passed into cloud as part of the 'msg' string, it prints out on the webpage UI. the Time which cloud understands, which user can not see.